### PR TITLE
Added consumes to EcalSeverityLevelESProducer

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

Also use ESGetToken gotten from consumes.

#### PR validation:

The code compiles.
This is a technical change and should not affect any results.

### CODE REVIEW

**Comment by slava77** [comment]
@smuzaffar @mrodozov 
did the code-checks get stuck?

**Comment by Dr15Jones** [comment]
code-checks

**Comment by smuzaffar** [comment]
@slava77 , yes it was stuck. Jenkins was not starting the machine to run the tests. I will look in to jenkins logs tomorrow to find out the reason.  I have started the machine and code checks are running now for all the pending PRs

**Comment by Dr15Jones** [comment]
please test

File: RecoLocalCalo/EcalRecAlgos/plugins/EcalSeverityLevelESProducer.cc [hunk 1]
```diff
@@ -32,13 +32,14 @@ class EcalSeverityLevelESProducer : public edm::ESProducer {
...
   edm::ReusableObjectHolder<HostType> holder_;
-  edm::ParameterSet pset_;
+  edm::ParameterSet const pset_;
+  edm::ESGetToken<EcalChannelStatus, EcalChannelStatusRcd> const channelToken_;
 };
 
-EcalSeverityLevelESProducer::EcalSeverityLevelESProducer(const edm::ParameterSet& iConfig) : pset_(iConfig) {
+EcalSeverityLevelESProducer::EcalSeverityLevelESProducer(const edm::ParameterSet& iConfig)
+    : pset_(iConfig), channelToken_(setWhatProduced(this).consumesFrom<EcalChannelStatus, EcalChannelStatusRcd>()) {
   //the following line is needed to tell the framework what
```
Comment (perrotta) [inline RecoLocalCalo/EcalRecAlgos/plugins/EcalSeverityLevelESProducer.cc]: These two comment lines could also get removed

Comment (Dr15Jones) [reply RecoLocalCalo/EcalRecAlgos/plugins/EcalSeverityLevelESProducer.cc]: done

**Comment by Dr15Jones** [comment]
please test

**Comment by perrotta** [comment]
+1
- Technical
- Jenkins tests pass

**Comment by kpedro88** [comment]
+1

**Comment by makortel** [comment]
part of #26748

**Pull request merged**